local_pathfinding/node_navigate.py

```diff
@@ -141,6 +154,35 @@ def get_desired_heading(self) -> float:
         # TODO: create function to compute the heading from current position to next local waypoint
         return 0.0
 
+    def LPath_callback(self):
+        """Get and publish the local path."""
+
+        waypoints = self.get_LPath()
+
+        local_path = Path(waypoints=waypoints)
+
+        msg = LPathData(local_path=local_path)
+
+        self.LPath_pub.publish(msg)
+        self.get_logger().debug(f"Publishing to {self.LPath_pub.topic}: {msg}")
+
+    def get_LPath(self):
+        """Get the local path.
+
+        Returns:
+            List[Tuple[float, float]]: The local path if all subscribers are active, else an empty
+                list.
+        """
+        if not self._all_subs_active():
+            self._log_inactive_subs_warning()
+            return -1.0
```

Comment:
Good idea to check for this error! What do we want the code to do in the case of inactive subs though? I tried running it, and it seems like if we return `-1.0` in place of a path, this causes a fatal erroranyway.

If we want this to stop code execution, maybe we should log it with a severe? Or if we want the code to keep running, maybe we should log a warning and return (0.0, 0.0) or something that should still be valid, but will look fishy?

Reply:
I will then return a HelperLatLon(0.0,0.0) when no subscribers are active.